src/SprykerSdk/SyncApi/OpenApi/Merger/PathsMerger.php

```diff
@@ -0,0 +1,439 @@
+    ): OpenApi {
+        $sourceOpenApiAsArray = json_decode(Writer::writeToJson($sourceOpenApi), true);
+
+        $parameterAsArray = $sourceOpenApiAsArray['components']['parameters'][$parameterName];
+
+        $refs = array_unique($this->getRefsFromArray($parameterAsArray, []));
+
+        foreach ($refs as $ref) {
+            $this->addInternalReference($targetOpenApi, $sourceOpenApi, $ref);
+        }
+
+        return $targetOpenApi;
+    }
+
+    /**
+     * @param \cebe\openapi\spec\OpenApi $targetOpenApi
+     * @param \cebe\openapi\spec\OpenApi $sourceOpenApi
+     * @param string $schemaName
+     *
+     * @return \cebe\openapi\spec\OpenApi
+     */
+    protected function addRefsFromSchema(
+        OpenApi $targetOpenApi,
+        OpenApi $sourceOpenApi,
+        string $schemaName
+    ): OpenApi {
+        $sourceOpenApiAsArray = json_decode(Writer::writeToJson($sourceOpenApi), true);
+
+        $schemaAsArray = $sourceOpenApiAsArray['components']['schemas'][$schemaName];
+
+        $refs = array_unique($this->getRefsFromArray($schemaAsArray, []));
+
+        foreach ($refs as $ref) {
+            $this->addInternalReference($targetOpenApi, $sourceOpenApi, $ref);
+        }
+
+        return $targetOpenApi;
+    }
+
+    /**
+     * @param array $array
+     * @param array $refs
+     *
+     * @return array
+     */
+    protected function getRefsFromArray(array $array, array $refs): array
+    {
+        foreach ($array as $key => $value) {
+            if ($key === '$ref') {
+                $refs[] = $value;
+            }
+
+            if (is_array($value)) {
+                $refs = $this->getRefsFromArray($value, $refs);
+            }
+        }
+
+        return $refs;
+    }
+
+    /**
+     * @param \cebe\openapi\spec\OpenApi $targetOpenApi
+     * @param \cebe\openapi\spec\OpenApi $sourceOpenApi
+     * @param string $reference
+     *
+     * @return void
+     */
+    protected function addInternalReference(OpenApi $targetOpenApi, OpenApi $sourceOpenApi, string $reference): void
+    {
+        if ($this->isExternalReference($reference)) {
+            return;
+        }
+
+        if ($this->isParameter($reference)) {
+            $this->addInternalParameter($targetOpenApi, $sourceOpenApi, $reference);
+        }
+
+        if ($this->isSchema($reference)) {
+            $this->addInternalSchema($targetOpenApi, $sourceOpenApi, $reference);
+        }
+    }
+
+    /**
+     * @param \cebe\openapi\spec\OpenApi $targetOpenApi
+     * @param \cebe\openapi\spec\OpenApi $sourceOpenApi
+     * @param string $reference
+     *
+     * @return \cebe\openapi\spec\OpenApi
+     */
+    protected function addInternalParameter(OpenApi $targetOpenApi, OpenApi $sourceOpenApi, string $reference): OpenApi
+    {
+        $parameterName = $this->getObjectName($reference);
+
+        $parameterItem = $this->getParameterByReference($sourceOpenApi, $parameterName);
+
+        $targetOpenApi->components->parameters = array_merge(
```

Comment:
This is one line PHPStan is complaining about and the complains are valid. You try to access `parameters` on `Component|null`

You need to do something like 

```
if (!is_null($targetOpenApi->components)) {
            $targetOpenApi->components->parameters = array_merge(
                $targetOpenApi->components->parameters,
                [$parameterName => $parameterItem],
            );
        }
```
Otherwise you will get an exception like `Trying to access $parameters on null` when compents is null and from the Cebe doc-blocks it can be null.

Please revert that change in your `phpstan.neon` file and fix your code around the mentioned findings.